pkg/lib/json/json.go

```diff
@@ -25,14 +25,22 @@ import (
 	"github.com/cortexlabs/cortex/pkg/lib/files"
 )
 
-func Marshal(obj interface{}) ([]byte, error) {
+func MarshalIndent(obj interface{}) ([]byte, error) {
 	jsonBytes, err := json.MarshalIndent(obj, "", "  ")
 	if err != nil {
 		return nil, errors.Wrap(err, errStrMarshalJSON)
 	}
 	return jsonBytes, nil
 }
 
+func Marshal(obj interface{}) ([]byte, error) {
```

Comment:
Because of this change, it will be necessary to search for all occurrences of `json.Marshal` (where `json` is our json package) and replace them with `MarshalIndent()`

Reply:
I did a pass where I was going to replace the previous occurrences of `json.Marshal` with `json.MarshalIndent` but I couldn't find any instances where they were user facing so I didn't update them.